src/index.ts

```diff
@@ -1,23 +1,25 @@
-let numero: number = 0;
+let cantidadPersonas: number = 270;
+let personas: number[] = new Array(cantidadPersonas);
 let menorValor: number = 18;
 let mayorValor: number = 40;
 let cantMenor: number = 0;
 let cantMayor: number = 0;
-let contador: number = 0;
-let persona: number;
 
-function cargarEdades(menorValor: number, mayorValor: number) {
-  numero = Math.floor(Math.random() * 100);
-  if (menorValor <= numero && numero <= mayorValor) {
-    persona = numero;
+function cargarEdades() {
+  for (let indice = 0; indice < cantidadPersonas; indice++) {
+    let edad =
+      Math.floor(Math.random() * (mayorValor - menorValor + 1)) + menorValor;
+    personas[indice] = edad;
   }
-  return persona;
 }
```

Comment:
Cambiamos la funcion **cargarEdades**, el objetivo de esta funcion va a ser llenar el array **personas** con valores aleatorios entre 18 (menorValor) y 40 (mayorValor).

Para esto agregamos un **for** en la función que va a repetirse 270 veces (cantidad de personas), y en cada vuelta va agregar una edad aleatoria entre 18 y 40 que se calcula de esta forma: `Math.floor(Math.random() * (mayorValor - menorValor + 1)) + menorValor;`